js/components/bottom-panel.js

```diff
@@ -23,7 +23,7 @@ function toggleFullscreen () {
 }
 
 @inject('simulationStore') @observer
-export default class BottomPanel extends PureComponent {
+export default class BottomPanel extends Component {
```

Comment:
Unrelated change, but the tests were complaining that this isn't a `PureComponent` any more since it has state.